src/tree-manager/index.js

```diff
@@ -50,7 +50,7 @@ class TreeManager {
     const matches = []
 
     const addOnMatch = node => {
-      if (node.label.toLowerCase().indexOf(searchTerm) >= 0) {
+      if ((node.label + ' ' + node.hiddenLabel).toLowerCase().indexOf(searchTerm) >= 0) {
```

Comment:
Shouldn't this be better off by matching on individual props rather than concatenating them, like:

```
if (node.label.toLowerCase().indexOf(searchTerm) >= 0 || node.hiddenLabel.toLowerCase().indexOf(searchTerm) >= 0)
```

It's a little more verbose but seems more true to its purpose and avoids creating extra strings

Reply:
It's a function now